_includes/footer.html

```diff
@@ -1,42 +1,39 @@
-
 <footer class="page-footer">
   <div class="container">
     <div class="row">
-      <div class="col l6 s12">
+      <div class="col-md-6 col-xs-12">
         <h5 class="white-text">viperdev</h5>
-        <p class="grey-text text-lighten-4">The Leading Edge Way to Improve Code Quality</p>
-
+        <!-- Adding closing tags for increase in readability, HTMLLintBear asks to avoid closing p tag -->
+        <p class="grey-text text-lighten-4">The Leading Edge Way to Improve Code Quality</p>  <!-- Ignore HTMLLintBear -->
         <br>
-        <span style="color:whitesmoke">Navigation</span>
-
+        <span class="footer-navigation">Navigation</span>
         <ul class="footer-nav-links">
           {% assign sorted_pages = site.pages | sort:"order" %}
           {% for node in sorted_pages %}
           {% if node.order %}
           {% if page.url == node.url %}
-          <li class="active"><a href="{{node.url | prepend: site.baseurl }}" class="active">{{node.title}}</a></li>
+          <!-- Adding closing tags for increase in readability -->
+          <li class="active"><a href="{{node.url | prepend: site.baseurl }}" class="active">{{node.title}}</a></li>  <!-- Ignore HTMLLintBear -->
           {% else %}
-          <li><a href="{{node.url | prepend: site.baseurl }}">{{node.title}}</a></li>
+          <!-- Adding closing tags for increase in readability -->
+          <li><a href="{{node.url | prepend: site.baseurl }}">{{node.title}}</a></li>  <!-- Ignore HTMLLintBear -->
           {% endif %}
           {% endif %}
-
           {% endfor %}
         </ul>
       </div>
-      <div class="col l4 offset-l2 s12">
+      <div class="col-md-4 col-md-offset-2 col-xs-12">
         <h5 class="white-text">Any Questions or Queries ?</h5>
-
-<div >
- <a href="https://gitter.im/viper-development/support" target="blank" class="mailbtn btn-large waves-effect grey-text text-lighten-4 activate-sidecar">Chat with us <span class=""><i class="mailicn fa fa-commenting" aria-hidden="true"></i>
-</span></a>
-</div>
-<div style="color: lightgrey; padding: 0.5em " class="or-text">OR</div>
-<div>
-  <a href="mailto:contact@viperdev.io" class="mailbtn btn-large waves-effect grey-text text-lighten-4" style="padding-left: 2.5em;">Drop us a <span class="mail-icon"> <i class="mailicn material-icons">email</i></span></a>
-</div>
-
+        <!-- HTMLLintBear acting weird even when div is properly indented -->
```

Comment:
not for me, just getting `|    | Protocol: Do not specify the protocol unless required: Remove the protocol "https:" from the url.` which sounds valid but quickly googling on stackoverflow leads to me not wanting this so put that class of errors in the ignore section as well